core.spec.md

```diff
@@ -0,0 +1,501 @@
+
+```graphql example -- Using the @core directive with a spec's default name
+schema @core(feature: "https://spec.example.com/example/v1.0") {
+  query: Query
+}
+
+type User {
+  name: String @example(data: ITEM)
+}
+
+# An enum used to provide structured data to the example spec.
+# It is prefixed with the name of the spec.
+enum example__Data {
+  ITEM
+}
+
+directive @example(data: example__Data) on FIELD
+```
+
+The prefix MUST NOT be elided within documentation; definitions of schema elements provided within the spec MUST include the default prefix.
+
+## Elements which must be prefixed
+
+Feature specs MUST prefix the following schema elements:
+  - the names of any object types, interfaces, unions, enums, or input object types defined by the feature
+  - the names of any fields the spec introduces on *foreign types* defined in a different feature
+  - the names of any arguments the spec introduces on *foreign directives and fields* defined by a different feature
+  - the names of any directives introduced in the spec, with the exception of the *root directive*, which must have the same name as the feature
+
+:::[example](prefixing.graphql) -- Prefixing
+
+# Versioning
+
+VersionTag : "v" Version
+
+Version : Major "." Minor
+
+Major : NumericIdentifier
+
+Minor : NumericIdentifier
+
+NumericIdentifier : "0"
+  | PositiveDigit Digit*
+
+Digit : "0" | PositiveDigit
+
+PositiveDigit : "1" | "2" | "3" | "4" | "5" | "6" | "7" | "8" | "9"
+
+Specs are versioned with a **subset** of a [Semantic Version Number](https://semver.org/spec/v2.0.0.html) containing only the major and minor parts. Thus, specifications SHOULD provide a version of the form {Major}.{Minor}, where both integers >= 0.
+
+```text example -- Valid version tags
+v2.2
+v1.0
+v1.1
+v0.1
+```
+
+As specified by semver, spec authors SHOULD increment the:
+
+{++
+
+- MAJOR version when you make incompatible API changes,
+- MINOR version when you add functionality in a backwards compatible manner
+
+++}
+
+Patch and pre-release qualifiers are judged to be not particularly meaningful in the context of core features, which are (by definition) interfaces rather than implementations. The patch component of a semver denotes a bug fix which is backwards compatible—that is, a change to the implementation which does not affect the interface. patch-level changes in the version of a spec denote wording clarifications which do not require implementation changes. As such, it is not important to track them for the purposes of version resolution.
+
+As with [semver](https://semver.org/spec/v2.0.0.html), the `0.x` version series is special: there is no expectation of compatibility between versions `0.x` and `0.y`. For example, a processor must not activate implementation `0.4` to satisfy a requested version of `0.2`.
+
+## Ordering
```

Comment:
SUBSTANTIVE (in a sense):I still don't really think this section, which does not impact processing, is in the spec. I know there's a reference to it in the next section, but I don't think there's any particular reason to think that targeting a low major version is particularly more useful than targeting a higher major version.